Update fbcs version v1.11

dbkreling opened the pull request:
Update fbc versions to v1.11

Signed-off-by: Daniel Kreling <dkreling@redhat.com>

dbkreling committed:
fbc: update fbc files for 1.11.0 release

The bundle image reference comes from the latest successful
stage release in Konflux

Signed-off-by: Daniel Kreling <dkreling@redhat.com>
Signed-off-by: Julien Ropé <jrope@redhat.com>

dbkreling committed:
fbc: apply changes from render.sh

Signed-off-by: Daniel Kreling <dkreling@redhat.com>
Signed-off-by: Julien Ropé <jrope@redhat.com>

dbkreling commented:
TODOs:
- Update the reference for the osc-operator-bundle image with the newest one, which is done after the 1.11 release is completed.
- Confirm the version pattern for this PR ("1.11" vs "1.11.0").

littlejawa commented:
> Confirm the version pattern for this PR ("1.11" vs "1.11.0").

Nope :) 
My comment about version strings is valid only for the label we put inside the images.
The catalog actually needs to list every single release we make. If we don't specify the 3rd digit here, it will be pretty hard to differentiate 1.11.0 from 1.11.1 (and more) from the customer' and support's point of view. And it doesn't bring anything to us, as we need to rebuild the catalogs anyway.

The version label in the image, on the other hand, is not used by anybody (that we know of) and visible only when you inspect the image. That's why we can afford to keep it vague, as it allows us to avoid rebuilding an image if it is only because of the 3rd digit version update.

littlejawa committed:
FBC: add serviceAccountName for 4.18 and 4.19 FBCs

For some reason, the line is missing compared to the other pipelines.
And we get build errors that relate to this (token authentication issues).

Signed-off-by: Julien Ropé <jrope@redhat.com>

littlejawa commented:
Enterprise Contract failure comes from the FIPS check, complaining about the missing openssl lib in the dm-verity image.
We know about this problem, we need to ignore this until it is properly addressed on the test side.

gkurz commented on fbc/v4.17/catalog-template.yaml:
```diff
@@ -56,6 +56,9 @@ entries:
         skipRange: '>=1.1.0 <1.10.3'
+      - name: sandboxed-containers-operator.v1.11.0
+        replaces: sandboxed-containers-operator.v1.10.3
+        skipRange: '>=1.1.0 <1.11'
```
It seems we always put the `.0` before. Any reason to not do it here ? Just asking per curiosity.

littlejawa replied:
Ah, that's a miss :-( Good catch.
I fixed it on the "name:" line, but missed the skipRange.

gkurz replied:
I just stumbled on your previous comment at https://github.com/openshift/sandboxed-containers-operator/pull/1401#issuecomment-3601898497 :smiley: . Always putting the patch level seems better indeed.

gkurz commented on fbc/v4.17/catalog-template.yaml:
```diff
@@ -103,6 +106,8 @@ entries:
   - image: registry.redhat.io/openshift-sandboxed-containers/osc-operator-bundle@sha256:4477810692429ac498b6ec1b295e6ead0f8fea25e91f6ddf17255d7e349b82e1
     schema: olm.bundle
+  - image: registry.redhat.io/openshift-sandboxed-containers/osc-operator-bundle@sha256:f63fe210b7c83db08358ae3696e95d27aadd079f3f1cc6c5b0c89c1959eadffc
+    schema: olm.bundle
```
Latest pushed bundle as expected :+1: .

gkurz reviewed (approved):
/lgtm

Thanks @dbkreling and @littlejawa !
